Changelog.md

```diff
@@ -1,11 +1,18 @@
 Drag and Drop XBlock changelog
 ==============================
 
+Version 4.0.0 (2023-01-31)
```

Comment:
Nit: please update the version here too. Also, we could use `3.1.0` instead of `3.0.1`. After all, we are removing Maple compatibility.